packages/preview/gloss-awe-0.0.3/README.md

```diff
@@ -0,0 +1,123 @@
+# typst-glossary
+
+Automatically create a glossary in [typst](https://typst.app/).
+
+This typst component creates a glossary page from a given pool of potential glossary
+entries using only those entries, that are marked with the `gls` or `gls-add` functions in
+the document. See sample-usage document for details.
+
+⚠️ Typst is in beta and evolving, and this package evolves with it. As a result, no
+backward compatibility is guaranteed yet. Also, the package itself is under development
+and fine-tuning.
+
+## Marking the Entries
+
+To include a term into the glossary, it can be marked with the `gls` function. The
```

Comment:
I think this README would benefit a lot from one integrated example where the glossary pool is maybe also in the main file. I played with it a bit and it took some a bit of time to figure out how the pieces fit together.

Reply:
I agree, and added a complete sample and restructured the readme accordingly.